code/game/objects/items/weapons/RPD.dm

```diff
@@ -476,6 +476,8 @@ var/global/list/RPD_recipes=list(
 		return 0
 	if(istype(A,/area/shuttle)||istype(A,/turf/space/transit))
 		return 0
+	if(istype(A, /obj/structure/lattice))
```

Comment:
I'm somewhat confused as to what you mean.

Do you mean adding a return attackby of the space turf if nothing context sensitive affects the lattice attackby?

Space turfs already check for a lattice when you attack them, this modifies what happens when  you attack the lattice itself.